### PR TITLE
Updated with wishlist page

### PR DESCRIPTION
**Pull request by JaanviNL**
Screenshot (30)

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -74,6 +74,10 @@ position: relative;
     font-weight: bold;
     color: var(--basic-link);
 }
+.active-link
+{
+    color: var(--primary-btn);
+}
 
 .s-icon {
```
Comment (ranitasahaa1230) [inline index.css:82]: Can have better naming variables

File: navigation pages/wishist/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    color: var(--wishlist);
+}
+.products-card-w {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    row-gap: 2rem;
+     padding-left: 2rem;
+    margin: auto;
+    
+
```
Comment (ranitasahaa1230) [inline navigation pages/wishist/wishlist.css:15]: Remove all these extra spaces

**Review by ranitasahaa1230 — APPROVED** [review]
Looks good to merge

File: navigation pages/wishist/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+
+            <div class="wishlist-p">
+                <h2 class="center"> My Wishlist</h2>
+
+                <div class="products-card-w">
+
+                    <div class="product-card">
+                        <span><i class="fas fa fa-heart card-icon wishlist"></i></span>
+                        <img src="/assets/p-1.webp">
```
Comment (aurobindoGupta) [inline navigation pages/wishist/wishlist.html:44]: add alt

**Review by aurobindoGupta — APPROVED** [review]
nice work!!

**Pull request merged**